src/OrangePaymentSlipData.php

```diff
@@ -220,17 +253,26 @@ public function getCompleteReferenceNumber($formatted = true, $fillZeros = true)
             $completeReferenceNumber = str_pad($referenceNumber, 26, 'X', STR_PAD_LEFT);
         } elseif ($this->getWithBankingCustomerId()) {
             // Get reference number and fill with zeros
-            $referenceNumber = str_pad($referenceNumber, 20, '0', STR_PAD_LEFT);
+            $padLength = 26 - $this->bankingCustomerIdLength;
```

Comment:
We should not use magic numbers, I prefer to have a constant for that, as this number cannot be changed. E.g. ``static::REFNR_MAX_LENGTH``